Sources/Files.swift

```diff
@@ -298,6 +298,33 @@ public class FileSystem {
         return try! Folder(path: ProcessInfo.processInfo.homeFolderPath, using: fileManager)
     }
     
+    /// A reference to the document folder used by this file system. Returns nil on Linux by default.
+    public var documentFolder: Folder? {
+        #if os(Linux)
```

Comment:
How about simply making this API not available on Linux? Since trying to use it will just result in `nil` anyway. If you want, you can make an extension on `FileSystem` that has `#if os(Linux)` wrapped around it.